llvm/include/llvm/Analysis/InstSimplifyFolder.h

```diff
@@ -117,9 +117,10 @@ class InstSimplifyFolder final : public IRBuilderFolder {
     return simplifyCastInst(Op, V, DestTy, SQ);
   }
 
-  Value *FoldBinaryIntrinsic(Intrinsic::ID ID, Value *LHS, Value *RHS,
-                             Type *Ty) const override {
-    return simplifyBinaryIntrinsic(ID, Ty, LHS, RHS, SQ, nullptr);
+  Value *FoldBinaryIntrinsic(Intrinsic::ID ID, Value *LHS, Value *RHS, Type *Ty,
+                             Instruction *FMFSource = nullptr) const override {
```

Comment:
```suggestion
                             Instruction *FMFSource) const override {
```
Don't use default argument in an overridden virtual function.